hudi-common/src/main/java/org/apache/hudi/common/storage/StorageSchemes.java

```diff
@@ -36,6 +36,10 @@
   S3A("s3a", false), S3("s3", false),
   // Google Cloud Storage
   GCS("gs", false),
+  // Azure WASB
+  WASB("wasb", false), WASBS("wasbs", false),
+  // Azure ADLS
+  ADLS("adls", false),
```

Comment:
I am not familiar with azure myself. but is it `adl` or `adls` or we need both? 
https://hadoop.apache.org/docs/current/hadoop-azure-datalake/index.html

Reply:
You're right, ADL is secured by default. URI fixed